css-view-transitions-1/Overview.bs

```diff
@@ -1614,8 +1612,11 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. Let |hasActiveAnimations| be a boolean, initially false.
 
-		1. [=list/For each=] |element| of |transition|'s [=ViewTransition/transition root pseudo-element=]'s [=tree/inclusive descendants=]:
+		1. [=map/For each=] |transitionName| → |capturedElement| of |transition|'s [=ViewTransition/named elements=],
```

Comment:
Sorry for the confusion but now that dug through the spec to answer your question, step 8.8.3.1 [here](https://drafts.csswg.org/css-view-transitions-1/#style-transition-pseudo-elements-algorithm:~:text=If%20any%20flat%20tree%20ancestor%20of%20capturedElement%E2%80%99s%20new%20element%20skips%20its%20contents%2C%20or%20capturedElement%E2%80%99s%20new%20element%20is%20not%20rendered%2C%20then%20return%20failure.) seems like a better spot for this. We already check all the other per element constraints which abort the transition here.

Reply:
Done